content/docs/manuals/networking-internals/proxy-conversion.md

```diff
@@ -0,0 +1,256 @@
+---
+
+In order to make the service layer networking functional, a worker-level tunnel-proxy is transparently instantiated
+as part of the Oakestra network component. The following picture is an example of what happens in a worker node
+based on the IPv4 implementation of the Network Manager component. Note that
+**IPv4 and IPv6 work identically**. For simplicity, we will stick to IPv4 addresses in this walkthrough.
+
+## Example
+
+Let's assume that we have two worker nodes, namely Node 1 and Node 2, each executing two containers.
+The containers are instantiated and managed by the Node Engine, while
+the Network Manager, creates a network namespace for each container (the cloud surrounding the container),
+enabling the Virtual Layer abstraction.
+
+{{<svg "overlay-example" >}}
+
+The Service Layer abstraction is realized hierarchically with a mechanism of *route interest registration* and
+*proxy translation*. This section details the proxy translation that allows transparent
+conversion of Service IPs into Namespace IPs, therefore enabling transparent Virtual Layer &harr; Service Layer conversion.
+
+Following the example mentioned above, suppose we deployed services X1 and X3 using the following deployment descriptor.
+
+{{< details "Click here to view the deployment descriptor for Services X1 and X3" >}}
+
+```yaml
+{
+  "sla_version" : "v2.0",
+  "customerID" : "Admin",
+  "applications" : [
+    {
+      "applicationID" : "",
+      "application_name" : "X",
+      "application_namespace" : "default",
+      "application_desc" : "X application",
+      "microservices" : [
+        {
+          "microserviceID": "",
+          "microservice_name": "X1",
+          "microservice_namespace": "default",
+          "virtualization": "container",
+          "code": "docker.io/X/X1",
+          "addresses": {
+            "rr_ip": "10.30.0.1",
+            "rr_ip_v6": "fdff:2000::1"
+          },
+        },
+        {
+          "microserviceID": "",
+          "microservice_name": "X3",
+          "microservice_namespace": "default",
+          "virtualization": "container",
+          "code": "docker.io/X/X3",
+          "addresses": {
+            "rr_ip": "10.30.1.30",
+            "rr_ip_v6": "fdff:2000::30",
+          	},
+        }
+      ]
+    }
+  ]
+}
+```
+
+Therefore, we register into the platform two services, `X.default.X1.default` and `X.default.X3.default`.
+At deployment time, we request 2 instances of X1 (`X.default.X1.default.0` and `X.default.X1.default.1`)
+and one instance of X3 (`X.default.X3.default.0`). The scheduling decision places the instances as shown in
+the picture.
+From the deployment descriptor, we asked the platform to provision the Service IP `10.30.1.30` to X3 with Round
+Robin policy. Therefore, X1 will use this address to perform load-balanced requests toward X3.
+
+{{< /details >}}
+
+---
+
+Click on different steps to see what is happening behind-the-scenes.
+
+{{< tabs-icon "Requests" >}}
```

Comment:
Images are broken for all steps except 1

<img width="886" alt="Screenshot 2024-12-18 at 15 39 32" src="https://github.com/user-attachments/assets/fc3b0a4b-2419-4141-b4e8-10df02e38879" />

Reply:
Again, works on my end. Did you make sure to clear the cache und rebuild?